contracts/0.8.9/WithdrawalQueueBase.sol

```diff
@@ -54,15 +65,14 @@ abstract contract WithdrawalQueueBase {
         uint64 timestamp;
         /// @notice flag if the request was claimed
         bool claimed;
+
+        uint64 reportTimestamp;
```

Comment:
:small_airplane: can fit into the same 2-slots space if both timestamps are `uint40` (~35k years ahead capacity)